batcher/aligned-batcher/src/lib.rs

```diff
@@ -672,7 +672,20 @@ impl Batcher {
             return Ok(());
         };
 
-        if !self.check_min_balance(proofs_in_batch + 1, user_balance) {
+        let msg_max_fee = nonced_verification_data.max_fee;
+        let Some(user_min_fee) = batch_state_lock.get_user_min_fee(&addr).await else {
+            std::mem::drop(batch_state_lock);
+            send_message(
+                ws_conn_sink.clone(),
+                SubmitProofResponseMessage::InvalidNonce,
+            )
+            .await;
+            self.metrics.user_error(&["invalid_nonce", ""]);
+            return Ok(());
+        };
+
+        // We estimate the min balance to be the minimum balance needed to pay for the
```

Comment:
```suggestion
        // We estimate the min balance to be the minimum balance needed to pay for the
```

This comment is incommplete

Reply:
addressed!